subsys/nfc/lib/Kconfig

```diff
@@ -111,25 +111,25 @@ choice NFC_SWI
 
 config NFC_SWI0
 	bool "SWI0"
-	depends on HAS_HW_NRF_EGU0 || HAS_HW_NRF_SWI0
+	depends on HAS_HW_NRF_EGU0 || HAS_HW_NRF_SWI0 || SOC_SERIES_NRF54LX
```

Comment:
out of curiosity - where is NRF54H?

Reply:
The nRF54H also has swi, but the nRF54L has different numbering with additional "0". In the platform_internal_thread.c file there is appropriate ifdef.

Reply:
NRF54H has available peripherals with low numbers 0,1,2, etc. So previous conditions apply for it.